front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -55,29 +58,86 @@ const StdcmConfig = ({
   const loaderRef = useRef<HTMLDivElement>(null);
 
   const dispatch = useAppDispatch();
-  const { updateGridMarginAfter, updateGridMarginBefore, updateStdcmStandardAllowance } =
-    useOsrdConfActions() as StdcmConfSliceActions;
+  const {
+    updateGridMarginAfter,
+    updateGridMarginBefore,
+    updateStdcmStandardAllowance,
+    updateOriginArrivalType,
+    updateDestinationArrivalType,
+  } = useOsrdConfActions() as StdcmConfSliceActions;
+
+  const { getOriginV2, getDestinationV2 } = useOsrdConfSelectors();
+  const origin = useSelector(getOriginV2);
+  const destination = useSelector(getDestinationV2);
 
   const { pathfindingState } = usePathfindingV2(undefined, pathProperties);
 
+  const [formErrors, setFormErrors] = useState<StdcmConfigErrors | undefined>(undefined);
+
   const inputsProps = {
     disabled: isPending || retainedSimulationIndex > -1,
     selectedSimulation,
     setCurrentSimulationInputs,
   };
 
   const startSimulation = () => {
-    if (pathfindingState.done) {
+    if (pathfindingState.done && !formErrors) {
       launchStdcmRequest();
     }
   };
 
+  const removeOriginArrivalTime = () => {
+    dispatch(updateOriginArrivalType(ArrivalTimeTypes.ASAP));
+  };
+  const removeDestinationArrivalTime = () => {
+    dispatch(updateDestinationArrivalType(ArrivalTimeTypes.ASAP));
+  };
+
   useEffect(() => {
     if (isPending) {
       loaderRef?.current?.scrollIntoView({ behavior: 'smooth' });
     }
   }, [isPending]);
 
+  const checkErrors = () => {
+    if (pathfindingState.error) {
+      setFormErrors({ errorType: StdcmConfigErrorTypes.PATHFINDING_FAILED });
+    } else if (origin && destination) {
```

Comment:
I think it would be easier to do if you add early returns after each setFormErrors, what do you think ?